README.md

```diff
@@ -230,43 +281,83 @@ enclave [tlcc_enclave](tlcc_enclave) can be build manually.
 Follow the instructions in the corresponding directories.
 
 For the integration of the enclave code into a Fabric
-chaincode, please follow the instructions in [ecc](ecc) for the chaincode
-enclave and [tlcc](tlcc) for the ledger enclave.
+chaincode, please follow the instructions in [ecc/](ecc) for the chaincode
+enclave and [tlcc/](tlcc) for the ledger enclave.
 
 In order to run and deploy a chaincode enclave we need to build the enclave
-registry. See [ercc](ercc).
+registry. See [ercc/](ercc).
+
+Moreover, we provide a set of integration tests in [integration/](integration) to demonstrate Fabric Private Chaincode
+capabilities.
+
+
+#### Trouble shooting
+
+Building the project requires docker. We do not recommend to run `sudo make`
```

Comment:
Maybe backwards pointer to above docker section on how to setup docker to run as user rather than root.?

Reply:
added